tutor/src/screens/assignment-review/details.js

```diff
@@ -281,22 +281,22 @@ const TaskingDates = observer(({ tasking, title }) => {
             <dl>
                 <div>
                     <dt>Open date</dt>
-                    <dd>
+                    <dd class="open-date">
                         <div>{tasking.opensAt.format(dateFormat)}</div>
                         <div>{tasking.opensAt.format(timeFormat)}</div>
                     </dd>
                 </div>
                 <div>
                     <dt>Due date</dt>
-                    <dd>
+                    <dd class="due-date">
                         <div>{tasking.dueAt.format(dateFormat)}</div>
                         <div>{tasking.dueAt.format(timeFormat)}</div>
                     </dd>
                 </div>
                 {!tasking.plan.isEvent && (
                     <div>
                         <dt>Close date</dt>
-                        <dd>
+                        <dd class="close-date">
```

Comment:
if these classes are purely used as selectors in the tests, it'd be better to use a `[data-testid="close-date"]` so that nobody confuses it with styling